src/webui/webapplication.cpp

```diff
@@ -234,13 +226,13 @@ void WebApplication::action_query_torrents()
     CHECK_URI(0);
     const RawStringMap& gets = request().gets;
 
-    print(btjson::getTorrents( toUtf8(gets["filter"]),
-                               toUtf8(gets["category"]),
-                               toUtf8(gets["sort"]),
-                               toUtf8(gets["reverse"]) == "true",
-                               toUtf8(gets["limit"]).toInt(),
-                               toUtf8(gets["offset"]).toInt()),
-           Http::CONTENT_TYPE_JSON);
+    print(btjson::getTorrents(gets["filter"],
+                              QString::fromUtf8(gets["category"].constData(), gets["category"].length()),
```

Comment:
Damn, I wanted to say that everything is OK... Why do you complicate everything? Why are you not doing just `QString::fromUtf8(gets["category"])`?